### DISCUSSION TITLE
Sidebar shows "Project Manager: Help and Feedback" rather than main "Project Manager" view

### DISCUSSION THREAD

**Discussion by bkbelton** [alefragnani/vscode-project-manager#585] [Q&A]
With the update to 12.3, the Sidebar view isn't displaying the main content (favorites, link to settings etc). Is this expected? (macOS Monterey 12.2
![sidebar](https://user-images.githubusercontent.com/66647/153233779-49d323e4-8041-415e-b28d-693b1dff2cb4.jpeg)
)

**Comment by alefragnani** [comment]
Hi @bkbelton ,

The latest release of the extension is 12.5 (released two days ago) but I don't see how/why that update could interfere on the views appearance. BTW, the extension doesn't manage the views state/position, so I suppose it can be related to VS Code itself.

But, I see two alternatives to _rollback_ any changes that has been made.

* You could use `Project Manager: Focus on ... View` command, to **focus/display** the missing views (and identify _where_ they were
* You could use `View: Reset View Locations`, to **reset** all views, from all extensions. The downside of this alternative is that any customization you has made on view's location will be lost, and every extension will return the its default location

Hope this helps
